learningequality/kolibri-design-system
# Points icons

## Description



Adds the leaf icon and re-pre-compiles the icons and updates the rst definitions. This was required to fully remove the dependency on SVG Icon Inline Loader which included the huge Material Design icon package in its dependencies. KDS has it in the devDependencies and we specifically only bundle used icons.

The primary motivator for this was to ensure people don't run into timeout issues while running `yarn install`.

#### Issue addressed


Fixes Kolibri https://github.com/learningequality/kolibri/issues/7890

## Steps to test

Check the icons in the testing server blow.  Note that I've included them in the colored icons set even though we don't actually give it the color that is used for the fill.

## Reviewer guidance




## Comments
